vmaas/types.go

```diff
@@ -300,10 +303,11 @@ type Nevra struct {
 }
 
 type RepoDetailCommon struct {
-	Label      string `json:"label"`
-	Name       string `json:"name"`
-	Basearch   string `json:"basearch"`
-	Releasever string `json:"releasever"`
+	Label        string `json:"label"`
+	Name         string `json:"name"`
+	Basearch     string `json:"basearch"`
+	Releasever   string `json:"releasever"`
+	Organization string `json:"organization"`
```

Comment:
should we always return Organization for repos in hosted env?

Reply:
For Repos API I'd return Organization always for repo listing purposes (unlike Updates API where we don't really need it in the response). We might also need it in Engine vmaas-sync job to properly check needed re-evaluations for hosts.